### ISSUE TITLE
integration issue: failed to start pouch daemon in background

### ISSUE THREAD

**Issue by houstar** [alibaba/pouch#2444]
### Ⅰ. Issue Description
Integration test failed

### Ⅱ. Describe what happened
Pouch Git Commit ID: a8ee96efaf88eaa7
Failed to `make integration-test`
```
[root@ip-172-31-33-105 pouch]# make integration-test
integration-test
./hack/testing/run_daemon_integration.sh
stop local-persist volume plugin...
start local-persist volume plugin...
stop pouch daemon...
start pouch daemon...
failed to start pouch daemon in background!
make: *** [integration-test] Error 1
```

### Ⅲ. Describe what you expected to happen
Passed all Integration test.

### Ⅳ. How to reproduce it (as minimally and precisely as possible)

1. Download the Pouch source code
2. Build the source code with
* $make
* $make build-integration-test
* $make integration-test
3.

### Ⅵ. Environment:

- pouch version (use `pouch version`): 
- OS (e.g. from /etc/os-release):
```
[root@ip-172-31-33-105 pouch]# cat /etc/os-release
NAME="CentOS Linux"
VERSION="7 (Core)"
ID="centos"
ID_LIKE="rhel fedora"
VERSION_ID="7"
PRETTY_NAME="CentOS Linux 7 (Core)"
ANSI_COLOR="0;31"
CPE_NAME="cpe:/o:centos:centos:7"
HOME_URL="https://www.centos.org/"
BUG_REPORT_URL="https://bugs.centos.org/"

CENTOS_MANTISBT_PROJECT="CentOS-7"
CENTOS_MANTISBT_PROJECT_VERSION="7"
REDHAT_SUPPORT_PRODUCT="centos"
REDHAT_SUPPORT_PRODUCT_VERSION="7"
```

- Kernel (e.g. `uname -a`):
- Install tools:
```
Linux ip-172-31-33-105.us-west-2.compute.internal 3.10.0-693.5.2.el7.x86_64 #1 SMP Fri Oct 20 20:32:50 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux
```
- Others:

**Comment by fuweid** [comment]
```
./bin/pouchd-integration DEVEL --debug --enable-lxcfs --add-runtime runv=runv
```

hi @houstar , could you run this command in your pouch root dir and paste the result to me? Thanks

**Comment by houstar** [comment]
@fuweid 

```shell
^C[root@ip-172-31-33-105 pouch]# ./bin/pouchd-integration DEVEL --debug --enable-lxcfs --add-runtime runv=runv
INFO[0000] start daemon at debug level
DEBU[2018-11-08T07:21:58.02073156Z] daemon config: ({"volume-config":{},"network-config":{"bridge-config":{"mtu":1500,"iptables":true,"ipforward":true,"userland-proxy":false}},"cri-config":{"listen":"unix:///var/run/pouchcri.sock","network-plugin-bin-dir":"/opt/cni/bin","network-plugin-conf-dir":"/etc/cni/net.d","sandbox-image":"registry.cn-hangzhou.aliyuncs.com/google-containers/pause-amd64:3.0","cri-version":"v1alpha2","stream-server-port":"10010","cri-stats-collect-period":10},"listen":["unix:///var/run/pouchd.sock"],"debug":true,"containerd":"/var/run/containerd.sock","default-registry":"registry.hub.docker.com","default-registry-namespace":"library","home-dir":"/var/lib/pouch","TLS":{},"default-runtime":"runc","enable-lxcfs":true,"lxcfs":"/usr/local/bin/lxcfs","lxcfs-home":"/var/lib/lxcfs","config-file":"/etc/pouch/config.json","cgroup-parent":"default","pidfile":"/var/run/pouch.pid","default-log-config":{"Type":"json-file"},"registry-service":{},"oom-score-adjust":-500,"add-runtime":{"runv":{"path":"runv","runtimeArgs":null}},"default-namespace":"default"})
Error: /var/lib/lxcfs is not a mount point, please run " lxcfs /var/lib/lxcfs " before Pouchd
ERRO[2018-11-08T07:21:58.02450938Z] /var/lib/lxcfs is not a mount point, please run " lxcfs /var/lib/lxcfs " before Pouchd
```

I'd like to add build-daemon-integration/build-integration-test as Makefile's `make integration-test` dependencies.

**Comment by houstar** [comment]
lxcfs installed on my CentOS 7.4 but not yet automatically ran `lxcfs /var/lib/lxcfs`.

**Comment by houstar** [comment]
Fixed by PR #2417
